linkedevents/serializers.py

```diff
@@ -30,7 +30,21 @@ def __init__(self, *args, **kwargs):
             self.translated_fields = []
             return
 
-        self.translated_fields = trans_opts.fields.keys()
+        if (
+            meta_fields := getattr(self.Meta, "fields", None)
+        ) and meta_fields != "__all__":
```

Comment:
Can also use `rest_framework.serializers.ALL_FIELDS` here, but `"__all__"` is fine too.

Reply:
Changed as suggested